packages/@react-native-windows/cli/src/config/projectConfig.ts

```diff
@@ -39,6 +39,7 @@ opt  - Item is optional. If an override file exists, it MAY provide it. If no ov
     projectLang: string, // (auto) Language of the project, cpp or cs, determined from projectFile
     projectGuid: string, // (auto) Project identifier, determined from projectFile
   },
+  experimentalFeatures: Record<String, string> // (auto) Properties extracted from ExperimentalFeatures.props
```

Comment:
Iirc there's also a copy of this schema on the docs site. Reminder to update that if it was not on your radar.

Reply:
Docs update PR here: https://github.com/microsoft/react-native-windows-samples/pull/514